### DISCUSSION TITLE
青衣带什么音擎啊？

### DISCUSSION THREAD

**Discussion by 0build0** [share121/inter-knot#2991] [其他]
![1723914906236](https://github.com/user-attachments/assets/67c56c7d-cf3e-4044-91d9-7390ad5ef08c)
青衣带什么音擎比较好啊？

**Comment by miaochu1** [comment]
专武你指的拥有<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Comment by BSNHD** [comment]
专武、莱卡恩专武、高星级人为刀俎或安比专武
